components/progressbar/stories/template.js

```diff
@@ -53,28 +54,28 @@ export const Template = ({
 					...customClasses.reduce((a, c) => ({ ...a, [c]: true }), {}),
 				})}
 				style=${styleMap(styles)}
-				value="50"
+				value="${value}%"
 				role="progressbar"
-				aria-valuenow="50"
+				aria-valuenow="${value}%"
 				aria-valuemin="0"
 				aria-valuemax="100"
 			>
 				${FieldLabel({
 					...globals,
 					size: `${size}`,
-					label: "Loading",
+					label: `${label}`,
 					alignment: "",
```

Comment:
I'm confused why this is passing through the variables as string interpolated variables, same comment as above that since it's named the same thing and the prop you can just pass the prop name through.

Reply:
I needed the string interpolation for the second FieldLabel label: `${value}%`, to get the `%` but not the in the other cases. Got it updated.